tools/monorepo-utils/src/changelog/lib/github.ts

```diff
@@ -0,0 +1,143 @@
+/**
```

Comment:
I really, really like that you're including tests for `projects.ts`. Would be nice to have some for `github.ts` as well, especially seeing how it's heavily relying on regular expressions. Always nice to have those tested well, imho. 

This could also help enforce some more separation between the fetching and the parsing tasks.

Reply:
yes, good call. I added tests and reworked the function to have an improved regex

Reply:
Thanks so much!